src/test/regress/sql/multi_colocated_shard_rebalance.sql

```diff
@@ -159,6 +159,9 @@ WHERE
     AND sp.shardstate != 4
 ORDER BY s.shardid, sp.nodeport;
 
+-- moving the shard again is idempotent
+SELECT master_move_shard_placement(13000001, 'localhost', :worker_2_port, 'localhost', :worker_1_port, 'force_logical');
```

Comment:
Let's not introduce more usage of the `master_xxx` functions
```suggestion
SELECT citus_move_shard_placement(13000001, 'localhost', :worker_2_port, 'localhost', :worker_1_port, 'force_logical');
```

Reply:
updated